ivanti_security_controls/help.md

```diff
@@ -0,0 +1,234 @@
+# Description
+
+Ivanti Security Controls is a unified IT management platform used for managing and protecting through Patch Management, Application Control, and Asset Inventory functionality.
+
+# Key Features
+
+* Ability to retrieve Ivanti Security Controls known agents
+* Ability to check agent status
+
+# Requirements
+
+* Ivanti Security Controls host and API port (default: 3121)
+* Username and password of Windows account where Ivanti Security Controls is installed 
+* (Recommended) Ivanti Security Controls certificate in order to enforce certificate verification
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|credentials|credential_username_password|None|True|Username and password|None|None|
+|host|string|None|True|Enter the hostname|None|None|
+|port|integer|3121|True|Enter the port|None|None|
+|ssl_verify|boolean|True|True|Validate certificate|None|None|
+
```

Comment:
Can we get an Example Input section here - we don't generate these currently, will need to be done manually. hopefully regenerating automatically from spec next week.